spec/features/schemes_spec.rb

```diff
@@ -204,9 +204,14 @@
                 locations.each do |location|
                   expect(page).to have_content(location.id)
                   expect(page).to have_content(location.postcode)
-                  expect(page).to have_content(location.units)
-                  expect(page).to have_content(location.type_of_unit)
-                  expect(page).to have_content(location.startdate&.to_formatted_s(:govuk_date))
+                  if FeatureToggle.new_locations_table_layout_enabled?
+                    expect(page).to have_content(location.name)
+                    expect(page).to have_content("Active")
+                  else
+                    expect(page).to have_content(location.units)
+                    expect(page).to have_content(location.type_of_unit)
+                    expect(page).to have_content(location.startdate&.to_formatted_s(:govuk_date))
```

Comment:
Does this ever get executed?

Reply:
Cause I'd expect out feature toggle to always be on in test environments, unless we mock it

Reply:
So I think the `RAILS_ENV` for the tests would always be `test`, I think we set that in `/spec/rails_helper.rb`.

We could do something like this to mock `RAILS_ENV`
```
allow(ENV).to receive(:[])
allow(ENV).to receive(:[]).with("RAILS_ENV").and_return('production')
```

Or we could mock the `new_locations_table_layout_enabled?` behaviour I think by doing something like this:
```
allow(FeatureToggle).to receive(: new_locations_table_layout_enabled?).and_return(false)
```

Similar to what we're doing here:
https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/blob/main/spec/requests/sales_logs_controller_spec.rb#L31